### ISSUE TITLE
v-droppable missing $droptag & $dropdata

### ISSUE THREAD

**Issue by happyDemon** [iFgR/vue-dnd2#1]
as the title mentions, the original implementation made it easier to differentiate the drop callbacks

```html
...
<li v-repeat="listX"
    v-draggable="x: {index: $index, dragged: 'dragged'}"
    v-dropzone="x: sort(listX, $index, $droptag, $dropdata)">{{$value}}</li>
...
<li v-repeat="listY" v-draggable="y: {index: $index, dragged: 'dragged'}" v-dropzone="
      y: sort(listY, $index, $droptag, $dropdata),
      x: move(listX, listY, $index, $droptag, $dropdata)
    ">{{$value}}</li>
...
```

**Comment by happyDemon** [comment]
I noticed this is not possible with Vue2 since it's restricting directives more.
Would defining the directive multiple times be the alternative?

**Comment by fgr-araujo** [comment]
if I understood, you are trying to use vue-dnd2 into a list to reordering them and/or insert new a item from another one list, right?

**Comment by happyDemon** [comment]
That's correct, but I needed the drop event to have extra data (so I could register the index the droppable container).

in the previous version it was possible to add extra parameters to the callback, 
with the current version it's only possible to define a handler function.

I got it to work, but it's not pretty (rewrote the method where the function call happened and changed the syntax to `{handler: sortFunc, param: {...}}`. it works but it's not elegant.

by the way want me to submit a PR to fixup the droppable's update func? at the moment every time there's an update, extra drag events get registered, firing them X times instead of once.

**Comment by fgr-araujo** [comment]
You are right.
You PR will be very welcome!
